src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed } = todo;
+
+  return (
+    <>
+      <article
+        className={classNames('TodoInfo', { 'TodoInfo--completed': completed })}
+      >
+        <h2 className="TodoInfo__title">{title}</h2>
```

Comment:
Readable code > short code. This one is a suggestion
```suggestion
        <h2 className="TodoInfo__title">
          {title}
        </h2>
```